advanced/mathematical_optimization/index.rst

```diff
@@ -191,26 +207,11 @@ It combines a bracketing strategy with a parabolic approximation.
 
    - |1d_optim_4|
 
-::
-
-    >>> from scipy import optimize
-    >>> def f(x):
-    ...     return -np.exp(-(x - .7)**2)
-    >>> x_min = optimize.brent(f)  # It actually converges in 9 iterations!
-    >>> x_min #doctest: +ELLIPSIS
-    0.699999999...
-    >>> x_min - .7 #doctest: +ELLIPSIS
-    -2.1605...e-10
 
 .. note:: 
    
-   Brent's method can also be used for optimization *constrained to an
-   interval* using :func:`scipy.optimize.fminbound`
+   You can use different solvers using the parameter method.
```

Comment:
Maybe we want to keep and adapt the comment on "bounds": the minimize_scalar function has a "bounds" argument, doesn't it?

Reply:
I have re-added the note on bounds.